python/src/aica_api/client.py

```diff
@@ -139,6 +141,10 @@ def set_application(self, payload: str) -> requests.Response:
         :param payload: The filepath of an application on the AICA computer, or the application content as a
         YAML-formatted string
```

Comment:
nitpick: this can now be simplified, since it covers all cases
```suggestion
        :param payload: The filepath of an application or the application content as a YAML-formatted string
```

Conversely you could be more verbose, as you prefer: "The filepath of an application on the host computer or the AICA application container, or the application content ..."